xea-core/accounts/models.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth.models import User
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+
+class UserProfile(models.Model):
+    class Meta:
+        app_label = 'accounts'
+
+    # Gender strings
+    MALE = 'M'
+    FEMALE = 'F'
+    OTHER = 'O'
+
+    GENDER_CHOICES = ((MALE, 'Male'), (FEMALE, 'Female'), (OTHER, 'Other'))
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    gender = models.CharField(max_length=1, choices=GENDER_CHOICES, blank=True)
```

Comment:
¿Qué pasa con aquellos usuarios cuya identidad de género no pueda expresarse únicamente en términos de male/female (cis, trans, pan, two-spirit, queer...) ?

Related: http://abcnews.go.com/blogs/headlines/2014/02/heres-a-list-of-58-gender-options-for-facebook-users/ (no necesariamente una buena salida)

cc/ @castrinho8 @braisarias